### PR TITLE
CAF::Path: fix typo in file_exists() documentation

### PR DESCRIPTION
**Pull request by jouvin**
Originally states that it tests if a file exists and is a directory...

Contributes to #215

### CODE REVIEW

File: src/main/perl/Path.pm [hunk 1]
```diff
@@ -308,7 +308,7 @@ sub directory_exists
 
 =item file_exists
 
-Test if C<filename> exists ans is a directory.
+Test if C<filename> exists ans is not a directory.
```
Comment (stdweird) [inline src/main/perl/Path.pm]: `ans` -> `and`

Comment (stdweird) [inline src/main/perl/Path.pm]: actually, this should read `and is a file`.

**Comment by jrha** [comment]
@jouvin ping

**Comment by jouvin** [comment]
I am doing it right now!

**Comment by jrha** [comment]
Thanks! :grin:

**Comment by jouvin** [comment]
Done!

File: src/main/perl/Path.pm [hunk 2]
```diff
@@ -331,8 +333,8 @@ Test if C<path> exists.
 This is basically the perl builtin C<-e || -l>,
 wrapped in a method to allow unittesting.
 
-A broken symlink exists. As C<-e> follows a symlink,
-a broken symlink either exists with C<-l> or not.
+A broken symlink (symlink whose target doesn't
```
Comment (stdweird) [inline src/main/perl/Path.pm]: the proof after eating too much pudding:
```
[stdweird@gast039e tmp]$ ls -l /tmp/a
ls: cannot access '/tmp/a': No such file or directory
[stdweird@gast039e tmp]$ ls -l /tmp/b
lrwxrwxrwx 1 stdweird stdweird 1 Feb 28 17:48 /tmp/b -> a
[stdweird@gast039e tmp]$ PERL5LIB=/usr/lib/perl perl -e 'package a;use parent qw(CAF::FileReader CAF::Path);package main;$b=a->new("/tmp/x");print $b->directory_exists("/tmp/b"),"\n"'

[stdweird@gast039e tmp]$ PERL5LIB=/usr/lib/perl perl -e 'package a;use parent qw(CAF::FileReader CAF::Path);package main;$b=a->new("/tmp/x");print $b->file_exists("/tmp/b"),"\n"'

[stdweird@gast039e tmp]$ PERL5LIB=/usr/lib/perl perl -e 'package a;use parent qw(CAF::FileReader CAF::Path);package main;$b=a->new("/tmp/x");print $b->any_exists("/tmp/b"),"\n"'
1
```

**Pull request merged**